src/renderer/ui/components/generic/LyricsViewer.js

```diff
@@ -1,21 +1,31 @@
 import $ from 'jquery';
 import React, { Component, PropTypes } from 'react';
 import { findDOMNode } from 'react-dom';
-
+import { shell, remote } from 'electron';
 import { requireSettings } from './SettingsProvider';
 
-class LyricsViewer extends Component {
+export class LyricsViewerComponent extends Component {
```

Comment:
The default export is remaining untouched, so the modules that are using LyricsViewer won't break. This additional export of the component itself (not a HOC wrapped around it) is required in order to be able to call its method in test suite. To call class methods of component rendered with enzyme we need to call `component.instance()` to get the instance of its class. If we import the HOC exported by default, the call to `component.instance()` returns wrong object which doesn't have methods we want to test.

Reply:
I'd rather we don't test a component that isn't technically being rendered directly.  You can fetch the inner component to call it's method using the enzyme `find` method.

https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/find.md

Reply:
Got it, will try `find`